### ISSUE TITLE
"How to write a custom provisioner?" doc

### ISSUE THREAD

**Issue by mathieu-benoit** [score-spec/docs#121]
Based on this blog post, it would be great/better to have a dedicated doc entry: "How to write a custom provisioner?" by taking inspiration from this blog post: https://score.dev/blog/writing-a-custom-score-compose-provisioner-for-apache-kafka/.

Maybe just 1 doc for both `score-compose` and `score-k8s` because the concept are the same? As opposed to one doc for each?

**Comment by ankur0904** [comment]
@mathieu-benoit
Can I work on this issue and make the PR?

**Comment by mathieu-benoit** [comment]
Thanks @ankur0904 for raising your hand on this one.

Before you start, I would like to make sure @astromechza can chime in providing high level guidance here.

**Comment by astromechza** [comment]
I think one of the core maintainers should rather work on this since this is not just technical but also quite idealogical around the mission, vision, and history of Score.

**Comment by ankur0904** [comment]
Could I collaborate on this with the core members working on it?

**Comment by mathieu-benoit** [comment]
Yeah good point @astromechza , I'll assign this ticket to you and let's see from there.

@ankur0904, sorry but after some discussions, we need some refinements on this one (that could land to actually write the doc itself like @astromechza mentioned) and we don't think anymore that this one is a good candidate as a `good first issue`, I'll remove the labels.

**Comment by ankur0904** [comment]
Okay, thanks for the clarification 
@mathieu-benoit

**Comment by mathieu-benoit** [comment]
Based on this discussion https://github.com/score-spec/score-compose/issues/237#issuecomment-2600952254, tracking this here that we will also need to document the `class` aspect.
